### ISSUE TITLE
[feat]  <rh-footer> add font-size revert instead of px value for

### ISSUE THREAD

**Issue by heyMP** [RedHat-UX/red-hat-design-system#312]
### Description

rh-footer explicitly sets the font-size of the :host component to `18px`.  This was a bandaid to account for sites that had different root font-sizes.

### Suggested solution

We should do this instead:

```
What we came up with was reverting the font-size: revert on :host will set the components base back to the users setting.  I figure we can trap door this with var(--rh-component-base-font-size, revert) or whatever
```
cc: @zeroedin @markcaron

**Comment by heyMP** [comment]
@zeroedin is checking on the best corse of action on this.

**Comment by bennypowers** [comment]
do we need a variable for this? Users that want to change the default should just
```css
rh-footer {
  font-size: 25em; /* 🌶 */
}
```

**Comment by zeroedin** [comment]
The fix I worked in for `rh-secondary-nav` resets the font-size to user preference, then uses `em` to style component sizing.

```
:host {
  font-size: var(--rh-secondary-nav-base-font-size, initial);
}
```
This resets to initial but also allows a trap door if someone is adamant about setting that value.

As for your question benny, do we need a variable? No probably not.   Your solution is most likely more valid just override directly in light dom.

**Comment by bennypowers** [comment]
Closed by #406
